### ISSUE TITLE
jpg2dcm: support to split encapsulated JPEG/MPEG stream into multiple fragments

### ISSUE THREAD

**Issue by gunterze** [dcm4che/dcm4che#1269]
jpg2dcm: support to split encapsulated JPEG/MPEG stream into multiple fragments configurable by option

```
 -F,--fragment <length>   maximum length of one fragment. If the JPEG/MPEG
                          stream exceeds that value, it will be split into
                          several fragments, using a Fragmentable
                          Encapsulated Transfer Syntax; 4294967294
                          (=2^32-2) by default.
```

Related  #1243

**Comment by hczedik** [comment]
Hi @gunterze,
I am surprised you made the default so big (~4GiB).
This number cannot be represented in an (signed) int, which is still used for example in the BulkData class for length.
So doesn't supporting such a high default properly in dcm4che depend on the work I started in https://github.com/dcm4che/dcm4che/pull/1266 ?

**Comment by gunterze** [comment]
Fixed #1266 by https://github.com/dcm4che/dcm4che/commit/28a3a5b2a02eb96d920d80dab7bc670924a0f7b9
